processing/src/main/java/io/druid/segment/column/SimpleDictionaryEncodedColumn.java

```diff
@@ -71,7 +72,7 @@ public IndexedInts getMultiValueRow(int rowNum)
   @Override
   public String lookupName(int id)
   {
-    return cachedLookups.get(id);
+    return Strings.emptyToNull(cachedLookups.get(id));
```

Comment:
Totally agree with Xavier's comment.  Just want to add that I think we've worked around it at higher levels...